src/Helper/InflectsString.php

```diff
@@ -75,4 +77,15 @@ public function substr(string $string, int $start, ?int $length = null): string
 
         return substr($string, $start, $length);
     }
+
+    /**
+     * Translates a message using the translator.
+     */
+    public static function translate(string $text, array $args = []): string
```

Comment:
it could be a simple function `t()` or `__()` (not method) :)

Reply:
done

Reply:
oh ok but i had ~striked~ this one and left another note 😀

Reply:
https://github.com/adhocore/php-cli/pull/120#issuecomment-2509763155

Reply:
OK, now I understand better. I hadn't paid attention to the link you provided. I'm going to manage the argument positioning

So, in the case of the `t()` function, should we leave it or go back to the previous translate method?

Reply:
its ok as it is, we just need to consider the sprintf for wrapping up

Reply:
In this case, shouldn't positioning only be managed for texts with more than one argument?

For example, this is relevant for `The current (%d) is greater than the total (%d).` but much less so for `Command %s not found`. 

I would like to leave `Command %s not found` unchanged but change `The current (%d) is greater than the total (%d).` to `The current (%1$d) is greater than the total (%2$d).`.

Reply:
ya thats fine

Reply:
done